DigitalLearningSolutions.Data.Tests/Services/LearningHubSsoSecurityServiceTests.cs

```diff
@@ -154,6 +171,139 @@ public void VerifyHash_returns_false_for_hashes_created_outside_tolerance_time([
             result.Should().BeFalse();
         }
 
+        [Test]
+        public void ParseSsoAccountLinkingRequest_throws_exception_when_verifyHash_returns_false()
+        {
+            // Given
+            var linkLearningHubRequest = new LinkLearningHubRequest
+            {
+                Hash = "vrdz",
+                UserId = 56789,
+            };

```

Comment:
We should make the session identifier non-null so we know that's not causing the exception.

Reply:
I've removed that property as it's technically not part of the "incoming" request.